ext/js/data/options-util.js

```diff
@@ -792,6 +792,7 @@ class OptionsUtil {
         // Version 11 changes:
         //  Changed dictionaries to an array.
         //  Changed audio.customSourceUrl's {expression} marker to {term}.
+        //  Added anki.displayTags
```

Comment:
Another very minor nitpick: add a period at the end for consistency with other note comments.